strutil/chrorder.go

```diff
@@ -0,0 +1,21 @@
+// auto-generated, DO NOT EDIT!
```

Comment:
Should we commit this file to the tree? I was expecting not to.

Reply:
I think we should. The upside is that the code, as checked out, builds without any extra operations and any changes to the file are directly visible.